README.md

```diff
@@ -73,38 +127,59 @@ Finally, getting the result is done with the `result` endpoint:
 curl "http://localhost:8000/api/jobs/result/0"
 ```
 
-## Using the Geanlogos Web UI
+### `genealogos-frontend`
 Genealogos ships with a pure html/javascript web frontend.
 By default, this frontend uses `127.0.0.1` to connect to the `genealogos-api`.
 Changing this default can be done using the settings button in the top of the webpage.
 
 The Web UI currently only supports analyzing from a flake ref and attribute path, analyzing from a trace file is not yet supported.
 
-## Building Genealogos
-The easiest way to build Genealogos is using nix, simply call `nix build` to build the library or `nix build .#genealogos-cli` for the cli.
+## Contributing
+Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+If you have a suggestion that would make this better, please fork the repo and create a pull request. You can also simply open an issue.
 
-If you want to use cargo, you might run into [a cargo issue related to workspaces](https://github.com/rust-lang/cargo/issues/4463).
-As a workaround you can use [cargo-hack](https://github.com/taiki-e/cargo-hack), which is convienently provided in the `nix develop` shell.
-Instead of running `cargo [build,test,doc,etc]`, use `cargo hack [build,test,doc,etc]`.
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/amazing-feature`)
+3. Commit your Changes (`git commit -m 'feat: add some amazing-feature'`)
+4. Push to the Branch (`git push origin feature/amazing-feature`)
+5. Open a Pull Request
```

Comment:
Is this useful? I would assume that GitHub users are familiar with the PR workflow (or they don't, “Fork the Project” is already too much for them)

Reply:
Good point!